packages/cloud-cognitive/src/components/AddSelect/AddSelect.js

```diff
@@ -311,6 +349,7 @@ AddSelect.propTypes = {
   onSubmitButtonText: PropTypes.string,
   open: PropTypes.bool,
   removeIconDescription: PropTypes.string,
+  searchResultsLabel: PropTypes.string,
```

Comment:
Don't forget to add docgen comments for all of your props at some point for storybook